alert-common/src/main/java/com/synopsys/integration/alert/common/channel/IssueTrackerChannel.java

```diff
@@ -58,7 +58,8 @@ public final MessageResult sendMessage(DistributionEvent event) throws Integrati
 
     @Override
     public final String getDestinationName() {
-        return channelKey.getUniversalKey();
+        // Required to avoid conflicts when registering new JMS Listeners
```

Comment:
I believe this class will be deleted anyway, but I'd prefer a FIXME to flag this so it's easy to find when we're ready to drop support.